docs/SchemaMigrationGuide.md

```diff
@@ -1,98 +1,89 @@
 
-* Non Prod scenarios (dev/test)
+## 2. Upgrade via Schema migration tool
+Schema migration tool is a command-line utility that admins can run to perform schema migrations on demand.
 
-  Since its  no harm that the dev/test application take some extra time during startup and any error/issue can be identified at the same time(if it happens). Also, it will reduce the overhead of role assignments and running the Schema migration tool.
+Schema upgrade scripts would still be part of any upgrade package for the application but the scripts will not run automatically on the server startup. In order to perform the schema upgrade, the schema admin would need to run the schema migration tool.
 
-## Upgrade via Schema migration tool
-Schema upgrade scripts are part of any upgrade package for the application but the scripts will not run automatically on the application startup. In order to perform the schema upgrade, the admin/assigned roles should run the Schema migration tool.
+ - ### Prerequisites
 
-### Schema migration tool
-The Schema migration tool is the command line utility to perform schema upgrade on demand.
-The tool needs to know which version the databases were at in order to select the appropriate migration scripts to upgrade the schema. 
+    - The schema migration tool is available.
 
-#### Commands
-The following commands are available via the tool
+ - ### When might you choose this option
+    - Production environment
+ 
+        It is highly recommended to use schema migration tool for the production environment to avoid downtime. Automatic schema migrations can take a long time, often in proportion to the size of the database. For this reason, performing schema migrations on startup may not be a good option.
+ 
+    - Role-based security
+ 
+        Only Database/Schema Admin can run this tool to upgrade the schema. It allows you to setup proper roles and permissions for the application e.g. the application is assigned the role as only Database Reader/Writer.
 
-|Command|Description|Options|
-|--------|---|---|
-|current|Returns the current versions from the SchemaVersion table along with information on the instances using the given version|--server|
-|available|Returns the versions greater than or equal to the current version along with links to the T-SQL scripts for upgrades|--server|
-|apply|Applies the specified version(s) to the connection string supplied. Optionally can poll the FHIR server current version to apply multiple versions in sequence|--server,<br /> --connection-string,<br /> --next,<br /> --version,<br /> --latest,<br /> --force|
+- ### Schema migration tool
 
-#### Options 
+    The detailed information about this tool is provided [here](SchemaMigrationTool.md).
 
-#### --next
+ - ### Best practices for schema upgrade on Sql server
```

Comment:
Should this be its own top level section? I think the advice in it is relevant to both the automatic and manual upgrade.